Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1184,7 +1171,22 @@ def plotContinents(self,x1,x2,y1,y2,projection,wrap,tmpl):
       return {}
 
   def renderTemplate(self,tmpl,data,gm,taxis,zaxis):
-    tmpl.plot(self.canvas,data,gm,bg=self.bg)
+    ## ok first basic template stuff, let's store the displays
+    ## because we need to return actors for min/max/mean
+    displays = tmpl.plot(self.canvas,data,gm,bg=self.bg)
+    returned = {}
+    for d in displays:
+        if d is None:
+          continue
+        texts=d.backend.get("vtk_backend_text_actors",[])
+        for t in texts:
+          ## ok we had a text actor, let's see if it's min/max/mean
+          txt = t.GetInput()
+          s0=txt.split()[0]
```

Comment:
Is `txt`guaranteed to be non-empty? This will throw an exception if the text actor has an empty string.